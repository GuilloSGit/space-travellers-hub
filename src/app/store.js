import { configureStore } from '@reduxjs/toolkit';
import missionsReducer from '../redux/missions/missions';
import rocketReducer from '../redux/rockets/Rockets';

const store = configureStore({
  reducer: {
    missions: missionsReducer,
    rockets: rocketReducer,
  },
});

export default store;
