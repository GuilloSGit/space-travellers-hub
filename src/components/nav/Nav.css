header {
  display: flex;
  align-items: center;
  margin-bottom: 50px;
}

header > ul {
  display: flex;
}

header > ul > li {
  list-style-type: none;
  margin-right: 10px;
  font-size: 1.1rem;
}

h1 {
  flex: 1;
  font-size: 1.3rem;
}

.active {
  padding: 3px;
  border-bottom: 3px solid black;
}
