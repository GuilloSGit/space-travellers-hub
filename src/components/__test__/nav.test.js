import React from 'react';
import renderer from 'react-test-renderer';
import { HashRouter as Router } from 'react-router-dom';
import { Provider } from 'react-redux';
import store from '../../app/store';
import Nav from '../nav/nav';

describe('Navbar Component Test', () => {
  it('Renders NavBar', () => {
    const app = renderer
      .create(
        <Provider store={store}>
          <Router>
            <Nav />
          </Router>
        </Provider>,
      )
      .toJSON();
    expect(app).toMatchSnapshot();
  });
});