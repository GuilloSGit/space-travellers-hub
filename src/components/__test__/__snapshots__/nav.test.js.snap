// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar Component Test Renders NavBar 1`] = `
<header>
  <h1>
    Space Travelers' Hub
  </h1>
  <ul>
    <li>
      <a
        href="#/"
        onClick={[Function]}
      >
        Rockets
      </a>
    </li>
    <li>
      <a
        href="#/missions"
        onClick={[Function]}
      >
        Missions
      </a>
    </li>
    <li>
      <a
        href="#/profile"
        onClick={[Function]}
      >
        Profile
      </a>
    </li>
  </ul>
</header>
`;
